samza-test/src/test/java/org/apache/samza/test/processor/TestStreamProcessor.java

```diff
@@ -47,26 +49,14 @@
 import org.apache.samza.test.StandaloneIntegrationTestHarness;
 import org.apache.samza.test.StandaloneTestUtils;
 import org.junit.Assert;
-import org.junit.Before;
 import org.junit.Test;
+import scala.Option$;
 
 import static org.mockito.Matchers.anyObject;
-import static org.mockito.Mockito.doNothing;
-import static org.mockito.Mockito.mock;
-import static org.apache.samza.test.processor.IdentityStreamTask.endLatch;
+import static org.mockito.Mockito.*;
```

Comment:
Let's import only the required packages and avoid "*" ?

Reply:
done